src/main/java/org/assertj/core/groups/Tuple.java

```diff
@@ -12,45 +12,38 @@
  */
 package org.assertj.core.groups;
 
-import static java.util.Collections.addAll;
 import static org.assertj.core.configuration.ConfigurationProvider.CONFIGURATION_PROVIDER;
-import static org.assertj.core.util.Lists.newArrayList;
-import static org.assertj.core.util.Objects.areEqual;
 
+import java.util.Arrays;
 import java.util.List;
 
 public class Tuple {
 
-  private final List<Object> datas = newArrayList();
+  private final List<Object> values;
 
   public Tuple(Object... values) {
-    addAll(datas, values);
+    this.values = Arrays.asList(values);
```

Comment:
minor point: I usually prefer a static import of `org.assertj.core.util.Lists.list` for conciseness (feel free to ignore me).

Reply:
Fair point, I'll change it! Maybe when we move to java 9+ we could replace these util methods with the corresponding `of()` factories (e.g., `List.of()`)

Reply:
`List.of` is better than `asList` I sill find `list` to be better but I don't mind if people use `List.of`, what I'm not too fond of is `of` because it's not always obvious what is being built

Examples:
```
this.values = List.of(values);  
```
```
this.values = list(values);  
```
```
// is that a list, an array, a set  or something else?
this.values = of(values) ; 
```

Reply:
In that case not using the static import would be mandatory 🙂